broker/config/settings.yml

```diff
@@ -703,6 +703,10 @@ defaults: &defaults
       id: 'bc158c9a-7934-401e-94ab-057082a5073f'
       name: 'v1.0-xsmall'
       free: false
+      supported_operations:
```

Comment:
`async_ops_supporting_parallel_sync_ops` is what I could think of as a name.

Also, I think use of & and * is better so that u declare it once and use elsewhere.

Reply:
Also, flat list instead of having read, as we discussed.

Reply:
done